modules/ossm-tutorial-bookinfo-verify-install.adoc

```diff
@@ -13,10 +13,9 @@ To confirm that the sample Bookinfo application was successfully deployed, perfo
 .Prerequisites
 
 * {SMProductName} {SMProductVersion} installed.
```

Comment:
Let's delete the version variable here, I don't think it's important to specify which version of Service Mesh to have installed.

Reply:
Done